benchmarking/significance-testing/README.md

```
@@ -0,0 +1,64 @@
+
+# Run significance test between two BFCL configurations
+python significance_test.py \
+    --config-a /path/to/score/model_a \
+    --config-b /path/to/score/model_b \
+    --n-bootstrap 10000 \
+    --output results.json
+```
+
+## Example Output
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)